test/integration/identify_test.rb

```diff
@@ -0,0 +1,42 @@
+require 'test_helper'
+
+class IdentifyTest < ActionDispatch::IntegrationTest
+
+  include Oaisys::Engine.routes.url_helpers
+
+  setup do
+    @routes = Oaisys::Engine.routes
+  end
+
+  def test_identify_xml
+    get oaisys_path + '?verb=Identify', headers: { 'Accept' => 'application/xml' }
```

Comment:
Rather than munge in the param name and argument, it's probably getter to let Rails handle that via `get oaisys_path(verb: 'Identify')`?